code/api/src/main/java/org/apache/tamaya/spi/ConversionContext.java

```diff
@@ -145,7 +145,7 @@ public ConfigurationContext getConfigurationContext() {
         /** The injection target (only set with injection used). */
         private AnnotatedElement annotatedElement;
         /** The ordered list of formats tried. */
-        private final Set<String> supportedFormats = new HashSet<>();
+        private final List<String> supportedFormats = new ArrayList<>();
```

Comment:
would a `LinkedHashSet` work here?

Reply:
It would.  We'd also want to change https://github.com/apache/incubator-tamaya/pull/11/files#diff-0ff94b3abf9ec69ee193d9ecdaad4936R42 to make it consistent.